docs/index.asciidoc

```diff
@@ -99,6 +100,16 @@ output {
 
 Controls this plugin's compatibility with the {ecs-ref}[Elastic Common Schema (ECS)].
 
+[id="plugins-{type}s-{plugin}-base64_encoding"]
+===== `base64_encoding`
```

Comment:
I would prefer we avoid adding boolean options tied to the current implementation detail of there being two options, instead opting for a name that is descriptive of what the option controls and a value that is descriptive of what is being done. This allows us to leave room for additional options in the future, such as alternate encodings that may be introduced upstream.

In this case, something like:

`encoding` with values `base64` and `binary`.

Reply:
+1 on previous comment.  If you take this suggestion, the doc will need to be reworded. Please ping me and I'll review/retest doc then.   :-)

Reply:
Thanks for suggesting it. I've applied the hint, so it's ready for a review :-)